packages/build/src/plugins/system_log.ts

```diff
@@ -0,0 +1,37 @@
+import type { FeatureFlags } from '../core/feature_flags.js'
+import type { SystemLogger } from '../plugins_core/types.js'
+
+import type { ChildProcess } from './spawn.js'
+
+export const captureStandardError = (
+  childProcess: ChildProcess,
+  systemLog: SystemLogger,
+  eventName: string,
+  featureFlags: FeatureFlags,
+) => {
+  if (!featureFlags.netlify_build_plugin_system_log) {
```

Comment:
I failed to mention this on the last PR, but I think this flag should be called `netlify_build_plugin_system_log_errors` or similar. The current name makes it sound like it's related to https://github.com/netlify/build/pull/5391. Having said that, the flag is super short-lived, so it shouldn't matter too much.